src/analytics/lambdas/scan-metadata-workflow/store-metadata-into-ddb.ts

```diff
@@ -262,10 +165,105 @@ async function batchWriteIntoDDB(metadataItems: any[]) {
         [ddbTableName]: itemsChunk,
       },
     };
-    await ddbDocClient.send(new BatchWriteCommand(inputPara));
+    try {
+      await ddbDocClient.send(new BatchWriteCommand(inputPara));
+    } catch (error) {
+      // log error and inputPara
+      if (error instanceof Error) {
+        logger.error('Error when batch write into ddb.', error);
```

Comment:
Log as a single message with an object with multiple parameters

Reply:
Refactored it.